utils/marian_client.py

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env python3
+"""
+A client that connects to a Marian server and translates text interactively.
+Run `python utils.marian_client.py` and type a text to translate in the terminal
+Source: https://github.com/marian-nmt/marian-dev/blob/master/scripts/server/client_example.py
+"""
+
+
+from __future__ import division, print_function, unicode_literals
+
+import argparse
+import sys
+
+from websocket import create_connection
+
+if __name__ == "__main__":
+    # handle command-line options
+    parser = argparse.ArgumentParser()
```

Comment:
Suggestion: Passing in the docs here will display them via argparse.

```suggestion
    parser = argparse.ArgumentParser(
        description=__doc__,
        formatter_class=argparse.RawTextHelpFormatter,  # Preserves whitespace in the help text.
    )
```